__tests__/decorators.test.ts

```diff
@@ -74,6 +76,32 @@ describe('decorators', () => {
       multipleOpenAPIsWithMixedParam() {
         return
       }
+
+      @Get('/responseSchemaDefaults')
+      @ResponseSchema(ModelDto)
+      responseSchemaDefaults(@Param('userId') _userId: number) {
+        return
+      }
+
+      @Get('/responseSchemaOptions')
+      @ResponseSchema(ModelDto, {statusCode: 400, contentType: 'text/csv'})
+      responseSchemaOptions(@Param('userId') _userId: number) {
+        return
+      }
+
+      @Get('/responseSchemaDecorators')
+      @HttpCode(201)
+      @ContentType("application/pdf")
+      @ResponseSchema(ModelDto)
+      responseSchemaDecorators(@Param('userId') _userId: number) {
+        return
+      }
+
+      @Get('/responseSchemaArray')
+      @ResponseSchema(ModelDto, {isArray: true})
+      responseSchemaArray(@Param('userId') _userId: number) {
+        return
+      }
     }
 
```

Comment:
@epiphone hmm, not sure tbh. If I understand https://github.com/epiphone/class-validator-jsonschema/blob/master/src/decorators.ts correct,y JSONSchema will currently overwrite existing keys. So the only way to be "consistent" no matter the order in which the decorators are set, would be to be defensive in ResponseSchema and never overwrite. Is that what you ment?

Reply:
As of #10 JSONSchemas are now `_.merge`d together instead of overwriting each other, so what I wrote on the above comment should work as intended (i.e. produce a `SomeResponseObject` response schema with `description: 'some description'`).

You're right about the order and consistency. In case of duplicate keys, for example

```typescript
@JSONSchema({responses: {
  '200': {
    'content': {
      'application/json': {
        schema: {
          '$ref': '#/components/schemas/Pet'
        }
      }
    ]
  }
}})
@ResponseSchema(SomeResponseObject)
handler() { ... }
```
the JSONSchema decorator overwrites ResponseSchema, and vice versa if their order was reversed. I think this is fine as long as we make it clear in the documentation.